static/app/components/replays/canvasReplayerPlugin.tsx

```diff
@@ -296,11 +294,7 @@ export function CanvasReplayerPlugin(events: eventWithTime[]): ReplayPlugin {
         return;
       }
       const [event, replayer] = queueItem;
-      try {
-        processEvent(event, {replayer});
-      } catch (err) {
-        Sentry.captureException(err);
-      }
+      processEvent(event, {replayer}).catch(handleProcessEventError);
```

Comment:
Changed to use `.catch`, alternatively we could have stuck a `await`, but then we'd have to change the handler fn to be async -- it's more clear to use `.catch` here as we don't care about the resolved value of `processEvent`